app/views/search/_user_product_approval_link.html.haml

```diff
@@ -1,4 +1,4 @@
 -# TODO: I18n confirmation
 = link_to user.last_first_name,
-  public_send("new_facility_#{@product.class.name.downcase}_user_path", @facility, @product, user: user.id),
-  data: { confirm: "Are you sure you wish to authorize this user to order this #{@product.class.name.downcase}?" }
+  public_send("new_facility_#{@product.class.name.underscore}_user_path", @facility, @product, user: user.id),
+  data: { confirm: "Are you sure you wish to authorize this user to order this #{@product.class.name.underscore}?" }
```

Comment:
Good catches all around. I did a lot of find and replace for downcase -> underscore, but probably didn't find everything.